source/extensions/http/custom_response/redirect_policy/redirect_policy.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
+#include <limits>
```

Comment:
Are these new headers needed?

Reply:
Sorry, this was was needed before, but now anymore. Removed.